treeherder/model/sql/jobs.json

```diff
@@ -450,14 +438,10 @@
                             ROUND( STD(end_timestamp - start_timestamp) )
                             AS SIGNED) AS 'running_std',
 
-                          GROUP_CONCAT(start_timestamp - submit_timestamp SEPARATOR ',')
-                            AS 'pending_samples',
-
                           GROUP_CONCAT(end_timestamp - start_timestamp SEPARATOR ',')
                             AS 'running_samples'
                    FROM job
-                   WHERE submit_timestamp >= ? AND
-                         start_timestamp >= submit_timestamp AND
+                   WHERE start_timestamp >= ? AND
```

Comment:
Since we no longer care about the `submit_timestamp` (since it's not required to generate running eta), the time window of interest is shorter - ie since the latest start time, not latest submit time. 

Once we `s/submit_timestamp/start_timestamp/` in this WHERE clause, the resultant `start_timestamp >= submit_timestamp` is redundant and can be removed.